R/serializer-csv.R

```diff
@@ -0,0 +1,17 @@
+#' @rdname serializers
```

Comment:
Run `devtools::document()`, please. At least the `NAMESPACE` file should be updated

Reply:
@schloerke all good!